service/topology_coordinator.cc

```diff
@@ -3218,8 +3216,7 @@ future<> topology_coordinator::start_tablet_load_stats_refresher() {
         } catch (...) {
             rtlogger.warn("Found error while refreshing load stats for tablets: {}, retrying...", std::current_exception());
         }
-        auto refresh_interval = utils::get_local_injector().is_enabled("short_tablet_stats_refresh_interval") ?
-                std::chrono::seconds(1) : tablet_load_stats_refresh_interval;
+        auto refresh_interval = std::chrono::seconds(_db.get_config().tablet_load_stats_refresh_interval_in_seconds());
```

Comment:
It's better for topology_coordinator to have its own reference to config rather than piggying on the back of db.